app/models/user.py

```diff
@@ -64,6 +64,20 @@ class DonorLevel(db.Model):
     description = db.Column(db.Text)
     users = db.relationship('User', backref='donor_level', lazy='dynamic')
 
+    @staticmethod
+    def insert_donor_levels():
+        donor_levels = {
+            'Advocate',
+            'Supporter',
+            'Benefactor'
+        }
+        for dl in donor_levels:
```

Comment:
let's use a different variable that's not `dl`. It's kind of confusing with the `donor_level` one below